### PR TITLE
Source Google Ads: drop schema field that filters out the data from stream

### LINKED ISSUE #12999: Data missing in Google Ads latest vrsion for `ad_group_ad_report`
## Environment
- **Airbyte version**:  0.35.61-alpha
- **OS Version / Instance**: GCP
- **Deployment**: Docker
- **Source Connector and version**: Google Ads 0.1.37
- **Destination Connector and version**: Snowflake 0.4.24
- **Step where error happened**: Deploy / Sync job / Setup new connection / Update connector / Upgrade Airbyte

## Current Behavior
Latest version of Google Ads is not bringing all campaigns for `ad_group_ad_report`
This is what Google return from querying directly the data:
- Responsive search ad
- Non-skipable in-stream ad
- Discovery ad

But `ad_group_ad_report` only brings campaign of type
- Reponsive search ad

There isn't any info in docs or Google changelog saying something about filtering data from reports.
Using Version with Google v8 brings all records and campaigns.

## Expected Behavior
*Tell us what should happen.*

## Steps to Reproduce
1.
2.
3.

## Are you willing to submit a PR?

Remove this with your answer.

### PR DESCRIPTION
**Pull request by davydov-d**
## What
Fix to return the missing data in one of the streams

## How
Drop one of the fields that filters out the data

---

Once the field `segments.keyword.ad_group_criterion` was introduced in the `ad_group_ad_report` stream, we got couple of missing data reports.

What we have in the UI:
image

what we have in the stream (`cost_micros` aggregated):
image

what we have in the stream after dropping the field (cost_micros aggregated):
image

This behavior is explained in a tiny [paragraph](https://developers.google.com/google-ads/api/docs/reporting/segmentation#selectability_between_segments_and_metrics) deep inside the Google Ads docs.
On one hand, this is legitimate behavior (the field is also present in the migration from Adwords [guide](https://developers.google.com/google-ads/api/docs/migration/mapping#ad_group_performance)), on the other this is very unobvious and data is lost when you use the connector of version 0.1.36-39 compared to other versions.

### CODE REVIEW

**Review by lazebnyi — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_ad_report.json [hunk 1]
```diff
@@ -174,9 +174,6 @@
     "segments.day_of_week": {
       "type": ["null", "string"]
     },
-    "segments.keyword.ad_group_criterion": {
-      "type": ["null", "string"]
-    },
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_ad_report.json:179]: Can we add a warning somewhere for future reference people DON't add this field again? :D

**Review by bazarnov — APPROVED** [review]

**Review by girarda — APPROVED** [review]

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_ad_report.json (hunk 1 above)
Comment (davydov-d) [reply airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_ad_report.json:179]: done

**Pull request merged**